storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6421,7 +6420,7 @@ static bool xb_export_cfg_write_index_fields(
     mach_write_to_1(value, compression_type);
 
     if (fwrite(&value, 1, sizeof(uint8_t), file) != sizeof(uint8_t)) {
-      msg("xtrabackup: Error: writing compression type info.");
+      xb::info() << "Error writing compression type info.";
```

Comment:
```suggestion
      xb::error() << "writing compression type info.";
```

Reply:
same as above (better to retain error here)

Reply:
or change to "Couldn't write compression type info" may be?